pyproject.toml

```diff
@@ -75,8 +75,6 @@ recommended = [
     'matplotlib',
     'pillow',
     'plotly',
-]
-standard = [
     'watchfiles',
```

Comment:
I do like the idea of watchfiles as a little more as recommended. Either that or the `recommended` group should be trimmed down to basically just watchfiles.

Reply:
I have reverted those changes.